src/modules/user/UserPage.module.scss

```diff
@@ -39,6 +39,17 @@
   text-align: center;
 }
 
+.segment {
+  text-align: start;
+}
+
+.code {
+  display: inline !important;
+  padding: 2px !important;
+  min-height: auto !important;
+  margin-left: 8px !important;
```

Comment:
Why is this all so !important? ;-)

All joking aside, which styles are you trying to override here?

Reply:
It's the Segment component I use to display the register command as a code section. Although to be fair I'm abusing the segment component a little bit. I could just use a div and style it the way I want, that would probably be better.